.github/workflows/frontend-test.yml

```diff
@@ -1,16 +1,13 @@
 name: Run Frontend Tests
 
-on:
-  push:
-    branches:
-      - main
-  pull_request:
-    branches:
-      - main
+on: [push, pull_request]
```

Comment:
don't remove the `branches` part

also, [path is not added](https://github.com/hokageCV/grudu-list/pull/43#discussion_r1779754110)